backend/tournesol/admin.py

```diff
@@ -177,7 +187,17 @@ def get_poll_name(self, obj):
 
 @admin.register(ComparisonCriteriaScore)
 class ComparisonCriteriaScoreAdmin(admin.ModelAdmin):
-    pass
+    list_filter = (
+        "comparison__poll__name",
+    )
+    list_display = (
```

Comment:
In the previous interface we were also able to see the related comparison (user + entity A + entity B). This new `list_display` doesn't provide these information.

It's not a problem of course, but it may be more meaningful if we are able to see the comparison of each criterion. We can display those information by adding `comparison` to the `list_display` for instance. I still don't know if this will generate an extra SQL query by criterion, adding `comparison` might require to update the query set.

**edit** adding `comparison` to `list_display` is safe according to my PostgreSQL logs

What do you think @lfaucon ?

We also might want to display the state of the comparison public, or not, but this is clearly a new feature that doesn't need to be in this PR.